lib/js_of_ocaml/json.ml

```diff
@@ -71,4 +158,7 @@ let output_reviver _key (value : Unsafe.any) : Obj.t =
     Obj.repr (array [| 255; value##.lo; value##.mi; value##.hi |])
   else Obj.repr value
 
-let output obj = json##stringify_ obj (Js.wrap_callback output_reviver)
+let output obj =
+  match Sys.backend_type with
```

Comment:
We could use the following to control the implementation, it would allow to write tests checking the two implementations behave the same.

```ocaml
let use_native_stringify =
  ref (
    match Sys.backend_type with
      | Other "js_of_ocaml" -> true
      | Native | Bytecode | Other _ -> false)
```